examples/webpack/class-name-compression-map.json

```diff
@@ -0,0 +1,27 @@
+{
+  "1wyb12am": "a",
```

Comment:
Where/how does this map get generated?

Reply:
Good question! For now it's assumed that library consumers manually curate the map and pass it to the babel-plugin. We will later create an automatic method to generate the map by statically analysing the codebase, and the map should be periodically updated by some a build.